src/Files.App/Data/Items/LocationItem.cs

```diff
@@ -96,6 +97,23 @@ public virtual object ToolTip
 			}
 		}
 
+		private FrameworkElement? itemDecorator;
+		public FrameworkElement? ItemDecorator
+		{
+			get
+			{
+				if (Section == SectionType.Favorites)
+				{
+					itemDecorator ??= new OpacityIcon()
```

Comment:
I think we can use PathIcon here since it's only a single path (OpacityIcon is used when there are multiple paths and different colors for each path).

Reply:
I had to use opacity icon for the eject icon in the end (it's made up of multiple paths) so let's keep it as is for consistency.